### PR TITLE
refactor: check spelling

### PR DESCRIPTION
**Pull request by jsoref**
Closes # 

### Description

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/hoppscotch/commit/f49b95be73709c260b50df159943501389c89d78#commitcomment-69494797

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/hoppscotch/commit/ed64aa5fe1c7794d411c9c3984c91abe3fdeb19f

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: packages/hoppscotch-app/helpers/import-export/import/index.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ type HoppImporter<T, StepsType, Errors> = (
 /**
  * Definition for importers
  */
-type HoppImporterDefintion<T, Y, E> = {
+type HoppImporterDefinition<T, Y, E> = {
```
Comment (jsoref) [inline packages/hoppscotch-app/helpers/import-export/import/index.ts:19]: This is probably an API break

File: packages/hoppscotch-app/plugins/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # PLUGINS
 
-This directory contains Javascript plugins that you want to run before mounting the root Vue.js application.
```
Comment (jsoref) [inline packages/hoppscotch-app/plugins/README.md:3]: brand

File: packages/hoppscotch-js-sandbox/README.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ This package deals with providing a JavaScript sandbox for executing various sec
 
 ## How does this work?
 
-This package makes use of [quickjs-empscripten](https://www.npmjs.com/package/quickjs-emscripten) for building sandboxes for running external code on Hoppscotch.
+This package makes use of [quickjs-emscripten](https://www.npmjs.com/package/quickjs-emscripten) for building sandboxes for running external code on Hoppscotch.
```
Comment (jsoref) [inline packages/hoppscotch-js-sandbox/README.md:20]: brand

**Review by jsoref — COMMENTED** [review]
Corrections were automatically selected by Google Sheets.

(JavaScript was done manually...)

File: packages/hoppscotch-app/helpers/import-export/import/index.ts (hunk 1 above)
Comment (AndrewBastin) [reply packages/hoppscotch-app/helpers/import-export/import/index.ts:19]: I don't see anything breaking.

**Review by AndrewBastin — APPROVED** [review]
Nothing looks broken as far as I have seen and tested. Hence, giving my green light. 💯 

By the way, really awesome project you have going on there, these are really deep observations into the spellings. Hats of to your valiant efforts! 🤠

**Review by liyasthomas — APPROVED** [review]
LGTM. [check-spelling](https://github.com/marketplace/actions/check-spelling) action is really helpful, thank you for working on such an amazing project.

**Pull request merged**